guides/common/modules/proc_applying-remote-scap-resources-in-disconnected-environment.adoc

```diff
@@ -0,0 +1,48 @@
+[id="applying-remote-scap-resources-in-disconnected-environment_{context}"]
+= Applying Remote SCAP Resources in Disconnected Environment
+
+If your host does not have internet access, you can download remote resources and include them in scans as local files.
+
+.Prerequisites
+* You have deployed a compliance policy on your host.
+* At least one scan using this policy has been already performed on your host.
+* You have `root` access on your host.
+
+.Procedure
+. On your host, find out the local path to the SCAP data stream by viewing the SCAP client configuration:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# grep -A 2 '^[[:digit:]]\+:' /etc/foreman_scap_client/config.yaml 
+3:
+  :profile: xccdf_org.ssgproject.content_profile_stig
+  :content_path: */var/lib/openscap/content/_96c2a9d5278d5da905221bbb2dc61d0ace7ee3d97f021fccac994d26296d986d.xml_*
+----
+. On the host, examine the data stream to find out which missing resource you must download:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# oscap info /var/lib/openscap/content/_96c2a9d5278d5da905221bbb2dc61d0ace7ee3d97f021fccac994d26296d986d.xml_ 2>&1 | grep 'WARNING: Skipping'
+*WARNING: Skipping '_https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL8.xml.bz2_' file which is referenced from datastream*
+----
+. On an online machine, download the missing resource:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# wget -O _security-data-oval-com.redhat.rhsa-RHEL8.xml.bz2_ _https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL8.xml.bz2_
+----
++
+To name the downloaded file correctly, remove the protocol and hostname from the URL, and replace slashes `/` with dashes `-`.
+. Upload the file to your host and place it to the home directory of `root`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# mv _security-data-oval-com.redhat.rhsa-RHEL8.xml.bz2_ /root
```

Comment:
This will work for any RHEL version, because oscap either runs in /root as pwd, or has it hardcoded for lookup of local files. See https://github.com/theforeman/foreman_scap_client/blob/master/lib/foreman_scap_client/client.rb#L43

Reply:
Correct, the only issue would be in case you have old Satellite (foreman_scap_client 0.5.0 or older) and new RHEL 8.6+.

Reply:
What kind of issue?